index.html

```diff
@@ -53,15 +53,15 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title>CALCULATOR</title>
 
 </head>
 
 
 <body>
 
 
-  <header class="text-center title">CALCULATOR | YOSHITHA</header>
+  <header class="text-center title" style="font-style: italic bold" >CALCULATOR </header>
```

Comment:
Don't use internel css use externel css @Pragya2056

Reply:
I have corrected it.